src/sentry/seer/similarity/utils.py

```diff
@@ -159,6 +159,45 @@
         "java-spring",
         "java-spring-boot",
         "perl",
+        # Remaining platforms
+        "apple",
+        "apple-ios-profiling-onboarding-1-install",
+        "apple-ios-profiling-onboarding-4-upload",
+        "apple-macos",
+        "c",
+        "capacitor",
+        "cfml",
+        "cocoa",
+        "cocoa-objc",
+        "cocoa-swift",
+        "dotnet-aspnet",
+        "dotnet-aspnetcore",
+        "dotnet-awslambda",
+        "dotnet-gcpfunctions",
+        "dotnet-google-cloud-functions",
+        "dotnet-maui",
+        "dotnet-uwp",
+        "dotnet-winforms",
+        "dotnet-wpf",
+        "dotnet-xamarin",
+        "electron",
+        "elixir",
+        "javascript-nextjs",
+        "javascript-nuxt",
+        "javascript-solidstart",
+        "kotlin",
+        "minidump",
+        # "native", Holding off for now
+        # "native-qt",
+        "nintendo-switch",
+        "objc",
+        # "other" and null platform will be excluded for now
```

Comment:
We need to exclude the empty string, too, as well as `null`. Seems like the code does do that, but we should probably include at least a mention of it here.